apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -66,7 +66,7 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
       <div className="mt-12 flex flex-wrap items-center justify-between gap-x-4 gap-y-8">
         <h1 className="text-4xl font-semibold">Documents</h1>
 
-        <div className="flex flex-wrap gap-x-4 gap-y-6 overflow-hidden">
+        <div className="flex flex-wrap gap-x-4 gap-y-6 overflow-hidden p-1">
```

Comment:
The trick is to do a `-m-1 p-1` to offset the hidden overflow 😄

Reply:
@Mythie - Just added a commit now with the above 👍